composer.json

```diff
@@ -14,23 +14,23 @@
     ],
 
     "require": {
-        "php": ">=5.6"
+        "php": ">=7.2"
```

Comment:
I wonder if we should be more conservative here (i.e. `~7.2 | ~8.0`)

Reply:
I am happy to do that, because actually we have no idea if this code will work with PHP 9.0 !
I think someone told me not to do that in a previous review :)

Reply:
Done and rebased